packages/react/src/tests/MessageRepository.test.ts

```
@@ -0,0 +1,690 @@
+
+      // Verify structure is preserved
+      const messages = repository.getMessages();
+      expect(messages.map((m) => m.id)).toEqual([
+        "parent-id",
+        "child1-id",
+        "child2-id",
+      ]);
+
+      // Verify content was updated
+      const contentPart = messages[1]!.content[0];
+      expect(contentPart.type).toBe("text");
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)